lib/features/science_clubs_view/widgets/ensure_visible_tags.dart

```diff
@@ -46,10 +47,10 @@ class EnsureVisibleTags extends DualTextMaxLines {
         return Column(
           crossAxisAlignment: CrossAxisAlignment.stretch,
           children: [
-            RichText(
+            AutoSizeText.rich(
```

Comment:
its' actually a real funny combination that we do calculate the max lines with our logic and then apply scaling down but the results looks pretty ok